### PR TITLE
Refactor: Improve HTML Structure and Add Application Entry Point

### PR DESCRIPTION
**Pull request by YonLiud**
I've done a few changes that I thought might be contributional

* **HTML Refactoring:**
1) Removed all inline styles from the <body> and <canvas> elements in the index.html file.
2) Improved readability and maintainability by moving styling concerns to external CSS (styles.css).
3) Ensured a cleaner separation of content (HTML) and presentation (CSS), adhering to best practices.

* **Application Entry Point:**
This change enables the application to be run directly as a standalone script.
Like so:
```bash
cd app
python3 __init__.py
```

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -34,4 +32,6 @@ def home():
 
     return app
     
-    
\ No newline at end of file
+if __name__ == '__main__':
+    app = create_app()
+    app.run()
```
Comment (s1nisteR) [inline app/__init__.py]: use `debug=True` inside `run()` to make it work similar to the flask command

**Comment by s1nisteR** [comment]
Looks good to me. You should additionally modify the `README.md` .

Basically just change `flask --app app run --debug` to `python __init__.py` for now. Also mention that this command launches a development server and should not be used in production.

**Review by s1nisteR — APPROVED** [review]

**Review by s1nisteR — CHANGES_REQUESTED** [review]
I added a comment on where to make the changes.

**Review by s1nisteR — APPROVED** [review]

**Comment by YonLiud** [comment]
should be good to go by now

**Comment by s1nisteR** [comment]
Nvm, I'll modify the README. Everything looks okay. Merging. Nice Work!

**Comment by YonLiud** [comment]
no need to modify, already done it in [fbf1d08](https://github.com/omen-osdev/omen-demo/pull/4/commits/fbf1d08cc91e22eb44208a59fc8e9ccad4a6e805)

**Pull request merged**